dts/bindings/haptics/ti,drv2605.yaml

```diff
@@ -26,10 +27,29 @@ properties:
       - "8X"
       - "16X"
       - "DISABLED"
+    default: "4X"
   loop-gain:
     type: string
     enum:
       - "LOW"
       - "MEDIUM"
       - "HIGH"
       - "VERY_HIGH"
+    default: "MEDIUM"
+  rated-voltage:
+    type: int
+    default: 0x3f
+    description: |
+      Sets the reference voltage for full-scale output during closed-loop
+      operation.
+  overdrive-clamp:
+    type: int
+    default: 0x89
```

Comment:
I'm not sure it's desirable to have the raw values there, i.e it is very opaque and hard to understand how this is a "voltage"
Could this be expressed in mV and the actual values for OD_CLAMP etc. automatically computed? It seems to be how Linux does it. Also would be good to use same property names I guess (`vib-overdrive-mv`, `vib-rated-mv`)?

Reply:
yeah ideally the driver would do the calculations, they are a bit tricky though as the meaning changes depending on the mode

did not realize that there was a Linux driver though, may be worth checking it out

Reply:
``` c
/*
 * Rated and Overdriver Voltages:
 * Calculated using the formula r = v * 255 / 5.6
 * where r is what will be written to the register
 * and v is the rated or overdriver voltage of the actuator
 */
static int drv260x_calculate_voltage(unsigned int voltage)
{
	return (voltage * 255 / 5600);
}
```
From the Linux driver, I see this is how they're converting between raw codes and voltages. It's not clear to me from the datasheet where this comes from.
![image](https://github.com/user-attachments/assets/23b52a1f-a83d-4aab-b3bd-486c7a0a75bd)

The datasheet walks the user to a raw code based on the actuator's electrical characteristics rather than provide a direct conversion.

![image](https://github.com/user-attachments/assets/f1afcdb3-abd0-4c6c-b6b7-74e1742faa2f)
I do see this though. Playing pretend chip designer I guess all of these voltage thresholds are tied internally to the same reference, so it would make sense for `RATED_VOLTAGE` and `OD_CLAMP` to scale the same way.

Having worked with the author of the Linux driver, Dan, at TI I do feel confident that the equation from the c snippet is correct. I'll go ahead and convert these properties to natural units.

Reply:
thanks for doing the investigation work